src/devices/video/pc_vga.cpp

```diff
@@ -5350,38 +5349,41 @@ void ati_vga_device::ati_port_ext_w(offs_t offset, uint8_t data)
 				}
 			}
 			else
-				logerror("ATI: ATI33 write %02x\n",data);
+			{
+				LOGMASKED(LOG_ATI, "ATI: ATI33 write %02x\n",data);
+			}
 			break;
 		case 0x38:
 			ati.ext_reg[ati.ext_reg_select] = data & 0xef;
-			logerror("ATI: ATI38 write %02x\n",data);
+			LOGMASKED(LOG_ATI, "ATI: ATI38 write %02x\n",data);
 			break;
 		case 0x39:
 			ati.ext_reg[ati.ext_reg_select] = data & 0xfe;
-			logerror("ATI: ATI39 write %02x\n",data);
+			LOGMASKED(LOG_ATI, "ATI: ATI39 write %02x\n",data);
 			break;
 		case 0x3a:  // General purpose read-write bits
 			ati.ext_reg[ati.ext_reg_select] = data & 0x07;
-			logerror("ATI: ATI3A write %02x\n",data);
+			LOGMASKED(LOG_ATI, "ATI: ATI3A write %02x\n",data);
 			break;
 		case 0x3c:  // Reserved, should be 0
 			ati.ext_reg[ati.ext_reg_select] = 0;
-			logerror("ATI: ATI3C write %02x\n",data);
+			LOGMASKED(LOG_ATI, "ATI: ATI3C write %02x\n",data);
 			break;
 		case 0x3d:
 			ati.ext_reg[ati.ext_reg_select] = data & 0xfd;
-			logerror("ATI: ATI3D write %02x\n",data);
+			LOGMASKED(LOG_ATI, "ATI: ATI3D write %02x\n",data);
 			break;
 		case 0x3e:
 			ati.ext_reg[ati.ext_reg_select] = data & 0x1f;
-			logerror("ATI: ATI3E write %02x\n",data);
+			LOGMASKED(LOG_ATI, "ATI: ATI3E write %02x\n",data);
 			break;
 		case 0x3f:
 			ati.ext_reg[ati.ext_reg_select] = data & 0x0f;
-			logerror("ATI: ATI3F write %02x\n",data);
+			LOGMASKED(LOG_ATI, "ATI: ATI3F write %02x\n",data);
 			break;
 		default:
-			logerror("ATI: Extended VGA register 0x01CE index %02x write %02x\n",ati.ext_reg_select,data);
+			LOGMASKED(LOG_ATI, "ATI: Extended VGA register 0x01CE index %02x write %02x\n",ati.ext_reg_select,data);
```

Comment:
Probably better to leave `logerror` at least for the “reserved” register and the unhandled conditions.

Reply:
`LOG_ATI` is enabled by default, and there's a reasonable argument to be made for allowing to electively switch it off without switching off logging entirely.